### ISSUE TITLE
Suggestion: Support for both free and pro including both kits and cdn

### ISSUE THREAD

**Issue by tommykjensen** [NowComponents/fa-icon#1]
I don't know how much time I have due to extremely busy work schedule but am definately interested in a component like this. Will try to contribute.

It should be easy to transition from using the free to using the pro version regardless if using cdn or kits.

**Comment by Quixomatic** [comment]
Yeah I really like that idea too, I set up a fairly basic implementation that pulls from just the solid free icons for now. If someone has access to the pro icons they should give it a shot!

**Comment by Quixomatic** [comment]
@tommykjensen, I just submitted a PR that should allow us to easily import pro icons into the component without any issues.

**Comment by dorsy99** [comment]
@tommykjensen if you are happy with this, I can close this issue.
